### PR TITLE
Updated README.md on points that confused me

### PR DESCRIPTION
**Pull request by russellballestrini**
I've never used assume role on the CLI before so these additional words seemed to help my understanding.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,31 +8,51 @@ On OS X, the best way to get it is to use homebrew:
...
+aws_access_key_id = AKIMYFAKEEXAMPLE
+aws_secret_access_key = wJalrXUtnFEMI/K7MDENG/MYxFAKEYEXAMPLEKEY
+
 [profile stage]
+# Stage AWS Account.
 region = us-east-1
 role_arn = arn:aws:iam::1234:role/SuperUser
-mfa_serial = arn:aws:iam::5678:mfa/eric-holmes
-source_profile = default
+usermgtsource_profile = usermgt
```
Comment (ejholmes) [inline README.md]: I think this should be `source_profile = usermgt`.

File: README.md [hunk 2]
```diff
@@ -8,31 +8,51 @@ On OS X, the best way to get it is to use homebrew:
...
+Setup a profile for each role you would like to assume in `~/.aws/config`.
 
-**Example**
+**For example, `~/.aws/config`:**
 
 ```ini
+[profile usermgt]
+# IAM user management AWS Account.
+region = us-east-1
+aws_access_key_id = AKIMYFAKEEXAMPLE
```
Comment (ejholmes) [inline README.md]: Dunno if it's worth mentioning in this doc, but in general these would be put in `~/.aws/credentials`.

**Review by ejholmes — APPROVED** [review]
Looks clearer!

File: README.md (hunk 1 above)
Comment (russellballestrini) [reply README.md]: Ah, yes.

**Comment by russellballestrini** [comment]
@ejholmes please review again, I adjusted the README.md to use the `~/.aws/credentials` file.

**Review by ejholmes — APPROVED** [review]

**Comment by russellballestrini** [comment]
@ejholmes I don't have merge abilities on this repo.

**Comment by phobologic** [comment]
@russellballestrini turns out none of us had that permission!  I added the team as collaberators :)

**Comment by russellballestrini** [comment]
@phobologic thanks!

**Pull request merged**